### ISSUE TITLE
Question about reproducing Setting 2 on Synapse-CT/SABS with EXCLUDE_LABEL

### ISSUE THREAD

**Issue by LW-J-King** [taozh2017/UPRENet#3]
Dear authors,
Thank you very much for open-sourcing your code and the previous explanation regarding the experiment protocol of Setting 2.
I am attempting to fully reproduce the reported Setting 2 segmentation results on the Synapse-CT/SABS dataset, but my reproduced Dice scores are far lower than the values presented in your paper.
My reproduction setup
1.Model code
I used the untouched original baseline model file models/fewshot.py without adding any extra modules or making any model-side modifications.
2.Synapse-CT label mapping I followed
Spleen: label 1
Right Kidney (RK): label 2
Left Kidney (LK): label 3
Liver: label 6
3.Configurations I applied for Setting 2
Upper group test labels: test_label=[1,6]
Lower group test labels: test_label=[2,3]
I set exclude_label=[1,2,3,6] for training data filtering for both groups
I modified the slice exclusion logic to filter out any slice that contains at least one target organ label (OR logic) in dataloaders/datasets.py:
exclude_idx = np.full(gt.shape[0], False, dtype=bool)
for i in range(len(self.exclude_label)):
    exclude_idx = exclude_idx | (np.sum(gt == self.exclude_label[i], axis=(1, 2)) > 0)
exclude_idx = idx[exclude_idx]
My understanding from your earlier reply is that Setting 2 trains with pseudo-labels, while removing all slices that contain the target test organs from the training set.

Result comparison
My reproduced Dice (%) on Synapse-CT Setting 2
Organ	Dice (%)
Spleen	58.09
Liver	74.00
LK	        50.48
RK	         43.56
Mean	56.53
There is a large performance gap between my reproduced results and the paper’s metrics, so I would like to clarify several critical details of the exact Setting 2 experiment protocol for Synapse-CT/SABS:
For Synapse-CT/SABS Setting 2, should the exclude_label parameter be set to [1,2,3,6] for both the Upper organ group and Lower organ group?
For slice exclusion logic: should we discard slices containing any one of the target organ labels, or only slices that contain all labels inside exclude_label simultaneously?
Are there any other configuration differences between Setting 1 and Setting 2, such as the hyperparameters test_label, use_gt, n_part, supp_idx, or the dataset train/test split scheme?
Does the official Setting 2 result rely on the original unmodified code logic in dataloaders/datasets.py, or does it require custom edits to the slice filtering code like the snippet I shared above?
Could you share the exact complete training and testing command lines you ran to produce the published Synapse-CT/SABS Setting 2 results?
Thank you sincerely for your time and guidance. I look forward to your reply.

**Comment by LW-J-King** [comment]
Should we keep the original AND logic in dataloaders/datasets.py,
or should we modify it to OR logic?
